drivers/hmis/lib/hud/hud_lists.json

```diff
@@ -0,0 +1,2885 @@
+[
```

Comment:
This file was generated from https://github.com/greenriver/hmis-warehouse/blob/44d385fa7f1d60cb9c7693bdadb394297e0d94eb/lib/util/hud.rb#L108, with some tweaks.

Reply:
We should reverse engineer this function so the maps are sourced from the JSON, but I didn't do that here. @eanders lets chat about that if you think it's worth doing at the same time

Reply:
I think it's worth doing, but a second pass is totally fine.